app/repositories/SessionRepository.scala

```diff
@@ -97,23 +97,18 @@ class DefaultSessionRepository @Inject()(
     }
   }
 
-  override def remove(identifierType: IdentifierType): Future[Option[UserAnswers]] = {
+  def remove(identifierType: IdentifierType): Future[Option[UserAnswers]] = {
     identifierType match {
       case id: Authed => collection(authCollectionName).flatMap(_.findAndRemove(Json.obj("_id" -> id.internalId)).map(_.result[UserAnswers]))
       case id: UnAuthed => collection(unauthCollectionName).flatMap(_.findAndRemove(Json.obj("_id" -> id.sessionId)).map(_.result[UserAnswers]))
     }
   }
 
-}
-
-trait SessionRepository {
-
-  val started: Future[Unit]
-
-  def get(identifierType: IdentifierType): Future[Option[UserAnswers]]
-
-  def set(identifierType: IdentifierType, userAnswers: UserAnswers): Future[Boolean]
-
-  def remove(identifierType: IdentifierType): Future[Option[UserAnswers]]
+  def updateTimeToLive(id: Authed): Future[Boolean] = {
+    get(id).flatMap {
+      case Some(ua) => set(id, ua)
+      case _ => throw new Exception(s"UserAnswers not found")
```

Comment:
We should probably just return a false here and in the controller match on that and do the necessary interpretation of that false there